login/meson.build

```diff
@@ -44,15 +46,21 @@ glome_login = executable(
     'glome-login',
     [
         'main.c',
-        'config.c',
-        'config.h',
     ],
-    dependencies : confuse_dep,
     link_with : login_lib,
     include_directories : glome_incdir,
     install : true,
     install_dir : get_option('sbindir'))
 
+cc = meson.get_compiler('c')
+libpam = cc.find_library('pam')
+pam_glome = shared_library('pam_glome',
```

Comment:
I was a little tempted to say that this should be conditional. On the other hand I'm of the belief that in the public build glome-login should eventually die and only ship the PAM module. :)

Reply:
I suppose there is no harm in building both until the decision is made on glome-login.